Article.md

```diff
@@ -0,0 +1,178 @@
+# Plugins for smart contract devs building on NEAR
+
+Some common functionality is required for many smart contracts. Examples are temporarily pausing certain features, staging and deploying a new version of the contract, and restricting access to contract methods. While such functionality is out of scope for an SDK like `near-sdk-rs`, ideally it is not implemented anew for every smart contract.
+
+The most obvious benefits of an open-source library are reusability and the value it adds to the ecosystem. The smart contract features mentioned above can be tricky to implement and cumbersome to test. Without a library, developers might gloss over functionality that does not add business value but still is critical for security. As `near-plugins` is open-source and used by many developers, there are more engineering hours and eyeballs dedicated to it compared to a solution specific to a single smart contract.
+
+## Case study: A counter managing permissions with ACL
+
+Let’s look at a case study to see how `near-plugins` can be useful to smart contract developers. We are building a `Counter` which stores its current value and has methods to increment, decrement and reset the value. It is intentionally kept simple to allow us to focus on how `near-plugins` adds functionality. This is what the contract looks like prior to using any plugins:
+
+```rust
+#[near_bindgen]
+impl Counter {
+    #[init]
+    pub fn new() -> Self {
+        Self { value: 0 }
+    }
+
+    /// Anyone can retrieve the current value.
+    pub fn value(&self) -> i64 {
+        self.value
+    }
+
+    /// Increases the value of the counter by one.
+    pub fn increment(&mut self) {
+        self.value += 1;
+    }
+
+    /// Decreases the value of the counter by one.
+    pub fn decrement(&mut self) {
+        self.value -= 1;
+    }
+
+    /// Resets the value of the counter to zero.
+    pub fn reset(&mut self) {
+        self.value = 0;
+    }
+}
+```
+
+*The final version of the code is available [in this repository on github](https://github.com/mooori/counter-acl-example). The `Counter` example is inspired by [near-examples/counter-rust](https://github.com/near-examples/counter-rust).*
```

Comment:
Made the links relative in [1582d56](https://github.com/mooori/counter-acl-example/pull/1/commits/1582d565e8d8d623282a9b7ad5c2506055a0ff68).

I’ve created the repo in my personal github as I wasn’t sure were it should live, to avoid triggering some workflows in a potentially wrong org. Moving it to an org is fine for me. Especially if this should be an official example/reference moving it to an org would make sense.